.github/workflows/hil_tests.yml

```diff
@@ -10,48 +10,57 @@ on:
   workflow_dispatch:
 
 jobs:
-  hil_test_zephyr:
-    strategy:
-      fail-fast: false
-      matrix:
-        include:
-          - hil_board: esp32_devkitc_wrover
-            west_board: esp32_devkitc_wrover
-            manifest: .ci-west-zephyr.yml
-            binary_name: merged.bin
-            binary_blob: hal_espressif
-          - hil_board: mimxrt1024_evk
-            west_board: mimxrt1024_evk
-            manifest: .ci-west-zephyr.yml
-            binary_name: zephyr.hex
-          - hil_board: nrf52840dk
-            west_board: nrf52840dk_nrf52840
-            manifest: .ci-west-zephyr.yml
-            binary_name: zephyr.hex
-          - hil_board: nrf9160dk
-            west_board: nrf9160dk_nrf9160_ns
-            manifest: .ci-west-ncs.yml
-            binary_name: merged.hex
-    uses: ./.github/workflows/hil_test_zephyr.yml
-    with:
-      hil_board: ${{ matrix.hil_board }}
-      west_board: ${{ matrix.west_board }}
-      manifest: ${{ matrix.manifest }}
-      binary_name: ${{ matrix.binary_name}}
-      binary_blob: ${{ matrix.binary_blob }}
+  # hil_test_zephyr:
```

Comment:
Are these going to remain commented out for now?

Reply:
This is a draft PR, it's not ready for review yet. I've commented out tests to avoid congestion on the runners during development.